bciers/apps/registration/app/components/operations/registration/FacilityInformationForm.tsx

```diff
@@ -128,7 +128,18 @@ const FacilityInformationForm = ({
       // errors are handled in MultiStepBase
       const response = await actionHandler(endpoint, method, "", {
         body: JSON.stringify(body),
+      }).then((resolve) => {
+        if (resolve?.error) {
+          return { error: resolve.error };
+        } else {
+          // Using window.location.href instead of MutliStepBase router.push as there were rerender issues due to
+          // Facility Grid listerning to params change on this page which sometimes hijacked the route change
+          window.location.href = `${
+            window.location.origin
+          }/registration/register-an-operation/${operation}/${step + 1}`;
+        }
       });
```

Comment:
Generally it's always preferable to use Next routing for internal links however in this case it fixed routing issues so I think it's okay to use on this one special page!